documentation/Set-PnPListItem.md

```diff
@@ -283,6 +283,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Force
+
+Forces the update of the list item even if there are no value changes.
```

Comment:
Could you elaborate in this help text on why this would be useful, like you already explained in the issue? Will help others wondering why they would ever need this.

Reply:
how's that?

Reply:
Perfect!